packages/utils/index.ts

```diff
@@ -1,7 +1,7 @@
 import type { PackageManifest } from "query-registry";
 
 const githubUrlRegex =
-  /(?:git\+)?https?:\/\/github\.com\/([^\/]+)\/([^\/]+)\.git/; // TODO: Don't trust this, it's chatgbd :)
+  /^(?:git\+)?https?:\/\/github\.com\/([^\/]+)\/([^\/]+)\.git$/;
```

Comment:
i didn't trust it 👀 

it would support trailing/leading characters before, which doesn't seem right. we grab the repo URL from the `repository` in `package.json`, so it should always be exact

so here i added `^` and `$` such that we don't match things like `blerghblarghhttps://github.com/foo/bar` anymore